odoo/tools/mail.py

```diff
@@ -298,6 +298,24 @@ def validate_url(url):
     return url
 
 
+def get_title_from_html(html_content, encoding='utf-8'):
+    try:
+        parser = etree.HTMLParser(encoding=encoding)
+        tree = html.fromstring(html_content, parser=parser)
+        title = tree.xpath('//meta[@property="og:title"]/@content')
+        return title[0] if title else tree.find('.//title').text
+    except Exception:
+        return False
```

Comment:
> This is not an RPC method, it had no reason to return a boolean in case of failure.

I think it's because it's used in the create values, so I think None is better than False (or we cast None to False in the caller, it can also work)